content/en/integrations/faq/how-to-import-datadog-resources-into-terraform.md

```diff
@@ -77,6 +77,21 @@ All example commands require the `--api-key` and `--app-key` flags.
 * Import all monitors and dashboards: `terraformer import datadog --resources=monitor,dashboard`
 * Import monitor with id 1234 and dashboard with id abc-def-ghi: `terraformer import datadog --resources=monitor,dashboard --filter=monitor=1234,dashboard=abc-def-ghi`
 
+## Generating resources with Terraform v0.13+
+
+Terraformer currently generates `tf`/`json` and `tfstate` files using Terraform `v0.12.29`. To ensure compatibility, run the upgrade command `terraform 0.13upgrade .` using Terraform `v0.13.x`. See [official Terraform docs][4] for upgrading.
```

Comment:
Would it make sense to put the version of terraformer next to `currently`?
Maybe something like: `As of version X.Y.Z, terraformer generates...`